barretenberg/cpp/scripts/barretenberg_module_digraph.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+set -eu
+
+TMP=tmp.dot;
+RESULT_DOT=barretenberg_modules.dot;
+RESULT_PNG=barretenberg_modules.png;
+
+# initialize a directed graph for graphviz
+echo digraph BarretenbergModules { > $TMP;
```

Comment:
BASH does not need a final semicolon, only if you want to do two lines on one line

Reply:
Indeed the semicolon finger fires automatically for me now;